report/test/renderer/report-renderer-test.js

```diff
@@ -288,7 +280,8 @@ describe('ReportRenderer', () => {
       .filter(url => DOCS_ORIGINS.includes(url.origin))
       .map(url => url.searchParams.get('utm_medium'));
 
-    assert.ok(utmChannels.length > 100);
+    console.log(utmChannels.length);
+    assert.ok(utmChannels.length >= 93);
```

Comment:
Could we do `> 75` or something? Seems like this number might need some flexibility in the future.